components/forms/ltft/LtftFormView.tsx

```diff
@@ -115,6 +117,24 @@ export const LtftFormView = () => {
   if (ltftStatus === "succeeded" || canEditStatus)
     return (
       <LtftViewWrapper>
+        <Button
+          data-cy="savePdfBtn"
+          disabled={!formData.id}
+          onClick={() => {
+            downloadLtftPdf(formData.id ?? "");
+          }}
+        >
+          Save a copy as a PDF
+        </Button>
+        {!formData.id ? (
+          <>
+            &nbsp;
+            <InfoTooltip
+              tooltipId={"pdfButtonInfo"}
+              content="The PDF download button would be disabled if the LTFT is unsaved. Please save before downdloading the PDF."
```

Comment:
Just wondering if we can simplify the messaging here?
Do we need the first sentence?
(If we do then the grammar probably needs tweaking.)
Happy to have a chat about this and any other comments. 😄

Reply:
It's a good shout. The first sentence is not necessary. 
Is the sentence "Please save before downloading the PDF." looks alright to you? (I also found the typo in the original sentence) Sorry my English always need advice.

Reply:
No worries, your English is infinitely better than my Chinese! Yes I think the second sentence looks fine. Very clear.